assets/js/blocks/products-by-attribute/block.tsx

```diff
@@ -26,9 +24,9 @@ import { getSetting } from '@woocommerce/settings';
 /**
  * Component to handle edit mode of "Products by Attribute".
  */
-class ProductsByAttributeBlock extends Component {
-	getInspectorControls() {
-		const { setAttributes } = this.props;
+const ProductsByAttributeBlock = ( props: Props ): JSX.Element => {
+	const getInspectorControls = () => {
```

Comment:
I'm thinking about the value of having these render methods and if it wouldn't be better to extract this one, `<ProductsByAttributeInspectorControls />` and `<ProductsByAttributeEditMode />` as separate components rather than methods I believe this will make the code easier to follow.